### PR TITLE
VLC and log file fixes

### PR DESCRIPTION
**Pull request by whirm**

### CODE REVIEW

File: Tribler/vlc.py [hunk 1]
```diff
@@ -49,7 +49,12 @@
 from inspect import getargspec
 
 __version__ = "N/A"
-build_date  = "Fri Jun 19 13:45:56 2015"
+build_date  = "Wed Sep 30 15:51:48 2015"
```
Comment (LipuFei) [inline Tribler/vlc.py]: @whirm They have a newer version here: https://github.com/oaubert/python-vlc/blob/master/generated/vlc.py
Built on Nov. 5.

Try it if this doesn't work.

**Comment by whirm** [comment]
I'll drop most of this PR's commits as I used a different approach  to fix the DLL loading issues.

**Comment by whirm** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Comment by whirm** [comment]
Merging as the test passed but failed due to an unrelated GUI unhandled exception. I'll open a new issue for that.

**Pull request merged**